lib/util/old_content.py

```diff
@@ -49,7 +49,7 @@ def _available_ssg_versions():
 def get_old_datastream():
     # installed SSG with datastream in /usr/share/xml
     installed = _installed_ssg_version()
-    root_datastream = util.find_datastream_in('/')
+    root_datastream = util.get_datastream('/')
```

Comment:
Honestly, I think calling `get_datastream()` here would be fine. But then `/` is nicely explicit for `root_datastream`. I guess either way works, I just wanted to give you a chance to change it.

Reply:
Yep, I wanted to have `/` explicit, so I kept it like that